### PR TITLE
Issue #2822 - Fix [Embedded content - reload the page to view]

### LINKED ISSUE #2822: Fix [Embedded content - reload the page to view]
I know there was issue #1290 opened about it but I cannot reopen it. Apparently nobody currently remembers why we remove media tags when the network stream is reloaded via Ajax. I want to take a stab at either fixing the original issue, or find out exactly why it was set like this in the first place.

### PR DESCRIPTION
**Pull request by Hypolite**
This PR adds a new user setting called "Bandwitdh Saver Mode" to control whether they want to display the [Embedded content - reload the page to view] on automatic update.

Fixes #2822 

I started converting the formatting to PSR-2 standards, make sure to ignore whitespaces when diffing.

### CODE REVIEW

**Comment by Hypolite** [comment]
Updated this PR to remove unwarranted formatting changes. I still couldn't help myself normalizing quotes and indenting whitespaces across `mod/update_*.php` files, removing extra lines, etc...

**Comment by annando** [comment]
What had you done with "mod/settings.php" that results in a complete replacement of the file?

**Comment by Hypolite** [comment]
The file had mixed indentation styles, I just normalized  it.

**Comment by Hypolite** [comment]
I left the default as what the current behavior is but I'd rather have it disabled by default.

**Comment by annando** [comment]
It seems as if you normalized it with spaces?

**Comment by Hypolite** [comment]
Good catch, I reverted this unwarranted formatting. There still is a lot going on.

**Comment by Hypolite** [comment]
There, no more trying to fix formatting in a hell hole of a PHP file. Can we please move on?

**Comment by Hypolite** [comment]
Thanks!
